test/evp_test.c

```diff
@@ -4148,6 +4148,24 @@ static int run_file_tests(int i)
     return c == 0;
 }
 
+/*
+ * Currently, EVP_<OBJ>_fetch doesn't support
+ * colon separated alternative names for lookup
+ * so add a test here to ensure that when one is provided
+ * libcrypo returns an error
+ */
+static int evp_test_name_parsing(void)
+{
+    EVP_MD *md;
+
+    if (TEST_ptr(md = EVP_MD_fetch(libctx, "SHA256:BogusName", propquery))) {
+        EVP_MD_free(md);
+        return 0;
+    }
+
+    return 1;
+}
+
```

Comment:
Please move this test to evp_extra_test2.c

Reply:
moved